ga4gh/cli.py

```diff
@@ -271,9 +274,12 @@ def server_main(parser=None):
     addGlobalOptions(parser)
     args = parser.parse_args()
     frontend.configure(args.config_file, args.config)
+    sslContext = None
+    if args.tls or ("OIDC_PROVIDER" in frontend.app.config):
+        sslContext = "adhoc"
     frontend.app.run(
-        host="0.0.0.0", port=args.port,
-        use_reloader=not args.dont_use_reloader)
+        host="localhost", port=args.port,
```

Comment:
Making host="localhost" is too restrictive here, as it means we can't use the server over the network. I agree it's better to have it default to localhost to prevent people accidentally deploying this development tool on the internet, but we should have a way of letting developers use the server over the network.